resources/programEnrollmentData.json

```diff
@@ -0,0 +1,81 @@
+{ "2014" : 
+    { "1" : 84,
+        "2" : 68,
+        "2T" : 8,
+        "3" : 54,
+        "4" : 21,
+        "5" : 17,
+        "6" : 4,
+        "7" : 1
+    },
+ "2015" : 
+    { "1" : 89,
+        "2" : 71,
+        "2T" : 7,
+        "3" : 55,
+        "4" : 43,
+        "5" : 17,
+        "6" : 3,
+        "7" : 1
+    },
+"2016" : 
+    { "1" : 111,
+        "2" : 89,
+        "2T" : 9,
+        "3" : 60,
+        "4" : 44,
+        "5" : 34,
+        "6" : 3,
+        "7" : 1
+    },
+"2017" : 
+    { "1" : 94,
+        "2" : 75,
+        "2T" : 10,
+        "3" : 63,
+        "4" : 48,
+        "5" : 35,
+        "6" : 7,
+        "7" : 1
+    },
+"2018" : 
+    { "1" : 128,
+        "2" : 102,
+        "2T" : 13,
+        "3" : 78,
+        "4" : 50,
+        "5" : 39,
+        "6" : 7,
+        "7" : 1
+    },
+"2019" : 
+    { "1" : 124,
+        "2" : 99,
+        "2T" : 17,
+        "3" : 68,
+        "4" : 63,
+        "5" : 40,
+        "6" : 8,
+        "7" : 1
+    },
+"2020" : 
+    { "1" : 112,
+        "2" : 90,
+        "2T" : 14,
+        "3" : 92,
+        "4" : 55,
+        "5" : 50,
+        "6" : 8,
+        "7" : 2
+    },
+"2021" : 
+    { "1" : 113,
+        "2" : 90,
+        "2T" : 15,
+        "3" : 93,
+        "4" : 74,
+        "5" : 44,
+        "6" : 10,
+        "7" : 2
+    }
+}
```

Comment:
historicCourseData.json and programEnrollmentData.json are both final version files for alg2. 
Everything else in resources/ is a mock for sprint 3 demo